packages/idyll-components/src/scroller.js

```diff
@@ -92,6 +94,12 @@ class Scroller extends React.Component {
     }
   }
 
+  handleStepProgress(response) {
+    const progress = { response };
+    const update = { progress: progress * 100 };
```

Comment:
Just a convention thing, but I would not multiply by 100, so the scroll progress is always between `[0, 1]`.